files/install-requirements.yaml

```diff
@@ -25,8 +29,10 @@
         - python3-bodhi-client
       state: present
 
-  - name: Install build RPM depoendencies
-    command: dnf -y builddep packit.spec
+  - name: Install packit-service sandbox for using run_command
+    pip:
+      name: git+https://github.com/packit-service/generator
+      executable: pip3
```

Comment:
Original purpose was to use rpm packages to avoid strange pip dependencies and troubles around that like ``bodhi client, libpagure`` , not something from pip what is not reliable.
It was also able to cover test when some deps is missing in specfile, but should be there.

Is there another reason of this change?

Reply:
This package will be only as a github repo. No RPM and it is not planned to package it.

Reply:
thanks, make sense